docs/topics/forms/media.txt

```diff
@@ -130,6 +132,21 @@ A tuple describing the required JavaScript files. See :ref:`the
 section on paths <form-asset-paths>` for details of how to specify
 paths to these files.
 
+``Script`` objects
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+.. class:: Script(src, **attributes)
+
+    :param src: String path to the JS file. See
+        :ref:`the section on paths <form-asset-paths>` for details.
+
+    :param attributes: HTML attributes to be set on the rendered ``<script>``
+        tag.
```

Comment:
```suggestion
    Represents a script file.

    The first parameter, ``src``, is the string path to the script file. See
    :ref:`the section on paths <form-asset-paths>` for details on how to
    specify paths to these files.

    The optional keyword arguments, ``**attributes``, are HTML attributes that
    are set on the rendered ``<script>`` tag.
```
I know I suggested the `:param` stuff, but it looks like it's not used very often. So maybe something like the above